docs/new-architecture-appendix.md

```diff
@@ -292,7 +307,69 @@ node node_modules/react-native/scripts/generate-specs-cli.js \
 
 In the above example, the code-gen script will generate several files: `MyLibSpecs.h` and `MyLibSpecs-generated.mm`, as well as a handful of `.h` and `.cpp` files, all located in the `ios` directory.
 
-## V. Note on Existing Apps
+## V. Codegen Naming Conventions
+
+### Native Components
```

Comment:
consider swapping Native Modules and Native Components to follow the same structure of the Website.